users/utils.py

```diff
@@ -0,0 +1,31 @@
+import jwt
+
+from django.http        import JsonResponse
+from .models            import User
+from config.settings    import SECRET_KEY, ALGORITHM
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            if 'Authorization' not in request.headers:
+                return JsonResponse({"message" : "NO AUTHORIZATION IN HEADER"}, status = 401)
+            
+            access_token = request.headers.get('Authorization')           
+            payload      = jwt.decode(access_token, SECRET_KEY, ALGORITHM)
+            user_id      = payload['user_id']
+         
+            if not User.objects.filter(id = user_id).exists():
+                return JsonResponse({"message" : "INVALID_UESR"}, status = 401)
+
+            user         = User.objects.get(id = user_id)
+            request.user = user
+
+            return func(self, request, *args, **kwargs)
```

Comment:
```suggestion

            request.user = User.objects.get(id = user_id)

            return func(self, request, *args, **kwargs)
```
user 변수에 굳이 안 담아도 될 것 같네요.

Reply:
request.user와 바로 연결했습니다.